sdk/storage/azure_storage_blob/tests/blob_container_client.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use azure_core_test::{recorded, TestContext};
+use azure_storage_blob::{clients::ContainerClient, BlobClientOptions};
+use std::{env, error::Error};
+
+#[recorded::test]
+async fn test_create_container(ctx: TestContext) -> Result<(), Box<dyn Error>> {
+    // Recording Setup
+    let recording = ctx.recording();
+    let mut options = BlobClientOptions::default();
+    recording.instrument(&mut options.client_options);
+
+    // Setup
+    let endpoint = format!(
+        "https://{}.blob.core.windows.net/",
+        recording.var("AZURE_STORAGE_ACCOUNT_NAME", None).as_str()
+    );
+
+    // Act
+    let container_client = ContainerClient::new(
+        &endpoint,
+        String::from("testcontainer11"),
+        recording.credential(),
+        Some(options),
+    )?;
+
+    // Assert
+    container_client.create_container(None).await?;
+
+    container_client.delete_container(None).await?;
```

Comment:
Why bother trying to clean up? I don't know why we ever did in .NET. Originally before I wrote the Test Resources scripts for all the lanugages, it was because we had long-lived test resources. No more. We provision as needed. That said, if you're running these again and again and trying to create an existing container fails, you could just decode to ignore *that* error. Otherwise, you're also just creating bigger session files that take longer to pull/push. Maybe have 1 test that tests creation and deletion, but ignore creation failures in the rest where it doesn't matter.

Reply:
Sure, we could decode and ignore that error. I will take that approach in the following revision.